zipkin-local.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+const _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
+const {
+  THEME_OPTIONS,
+} = require("gatsby-plugin-newrelic-test/utils/constants");
+const { getCiData } = require("./utils/getCiData");
+const ciAttributes = getCiData();
+exports.__esModule = true;
+exports.stop = exports.create = void 0;
+
+const _zipkin = _interopRequireDefault(require("zipkin"));
+
+const _zipkinTransportHttp = require("zipkin-transport-newrelic");
+
+const _zipkinJavascriptOpentracing = _interopRequireDefault(
+  require("zipkin-javascript-opentracing")
+);
+const _nodeFetch = _interopRequireDefault(require("node-fetch"));
+let logger;
+let recorder;
+/**
+ * Create and return an open-tracing compatible tracer. See
+ * https://github.com/opentracing/opentracing-javascript/blob/master/src/tracer.ts
+ */
+
+const create = () => {
+  logger = new _zipkinTransportHttp.HttpLogger({
```

Comment:
To promote function purity (see above).

```suggestion
  const logger = new _zipkinTransportHttp.HttpLogger({
```

Reply:
This comment still applies.